### PR TITLE
Added a lot of Marin splash text (syntax fixed)

### PR DESCRIPTION
**Pull request by muffinjets**

### CODE REVIEW

**Comment by foxsouns** [comment]
@muffinjets you're failing;

```
File "/home/runner/work/LADXR/LADXR/patches/aesthetics.py", line 202
    "Where do you live?" ask="Yes No",
    ^^^^^^^^^^^^^^^^^^^^^^^^
SyntaxError: invalid syntax. Perhaps you forgot a comma?
```

seems like maybe `"Where do you live?", ask="Yes No",` will work?

**Comment by muffinjets** [comment]
Ah, that would make sense, I think I just missed that comma.

**Comment by muffinjets** [comment]
Well I ran into a similar issue, I think having a Yes/No prompt is just not gonna happen there.  I was trying to reference this moment: https://i2.kym-cdn.com/photos/images/newsfeed/000/760/657/ec8.png

**Comment by foxsouns** [comment]
i think "Where did you come from?" would still be funny without the question, seeing as that is kind of the point of LADX: link is an outsider, and it isn't ever stated to the islanders that link arrived via boat from hyrule, so it checks out: not only in-canon, but as a joke, and as a reference, all in one.

**Comment by muffinjets** [comment]
I don't doubt that I'll make more of these at some point in the future, that'll be a good idea for next time

File: patches/aesthetics.py [hunk 1]
```diff
@@ -161,18 +161,95 @@ def reduceMessageLengths(rom, rnd):
...
+        "Keyboard compatible!",
+        "Teetsuuuuoooo!",
+        "Kaaneeeedaaaa!",
+        "Learn about allyship!",
+        "[This Marin text left intentionally blank.]",
+        "'Autological' is!",
+        "Technoblade never dies!",
+        "Thank you, CrystalSaver!",
+        "Wait, LADX has a rando?",
+        "https://tinyurl.com/ladxr",
```
Comment (daid) [inline patches/aesthetics.py:220]: The official URL is shorter then the tiny url: `http://ladxr.daid.eu/`

Comment (muffinjets) [reply patches/aesthetics.py:220]: I know, the two links are different though 😉

Comment (daid) [reply patches/aesthetics.py:220]: DAMN YOU. I'll allow it.

**Pull request merged**